### PR TITLE
Add assignment CH6

### PR DESCRIPTION
**Pull request by penny2243**
assignment CH6 reviewed by Hex school
wait for Camel review

### CODE REVIEW

**Comment by camel2243** [comment]
@penny2243 code pen 連結?

**Comment by penny2243** [comment]
切圖技巧 第六節作業 codepen:
https://codepen.io/penny2243/pen/djpEKQ

File: CH6/css/style.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
+body, html { 
+  font-family: "Microsoft JhengHei", "Segoe UI", sans-serif;
+}
+header {
+  width: inherit;
```
[the remark this answers is not in the document]
Comment (penny2243) [reply CH6/css/style.css]: 呃...不寫好像也可以啦XD

File: CH6/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>HTML、CSS教學</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css">
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <header class="clearfix">
+    <div class="contactErea">
```
[the remark this answers is not in the document]
Comment (penny2243) [reply CH6/index.html]: 想說老師叫大家用float那我也練習float XDD

[the remark this answers is not in the document]
Comment (camel2243) [reply CH6/index.html]: 喔好唷 👍

**Pull request merged**